syntax = "proto3";
package calendar_events;




import "easy_work_service_sdk/model/easy_work_service/calendar_events.proto";

/*
ListEvents请求
*/
message ListEventsRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    每页大小
    */
    int32 pageSize = 2 ;
    /*
    开始的日期 2020-02-02
    */
    string startDate = 3 ;
    /*
    保留字段，暂不使用
    */
    string toDate = 4 ;
    /*
    年
    */
    string year = 5 ;
    /*
    月 01 02 03 04 05 06 07 08 09 10 11 12
    */
    string month = 6 ;
}

/*
ListEvents返回
*/
message ListEventsResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated easy_work_service.CalendarEvents list = 4 ;
}

/*
ListEventsApi返回
*/
message ListEventsResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    calendar_events.ListEventsResponse data = 4 ;
}

